cmd/sbotcli/main.go

```diff
@@ -66,8 +66,13 @@ func init() {
 }
 
 var app = cli.App{
-	Name:    os.Args[0],
-	Usage:   "client for controlling Cryptoscope's SSB server",
+	Name:  os.Args[0],
+	Usage: "Client for interacting with a go-ssb server",
+	Description: `
+Please note, global options must be placed before sub-commands, e.g.
+
+    sbotcli --key <@...ed25519>" <cmd> <args>
```

Comment:
Looks like an errant `"` after `<@...ed25519>`?

Reply:
Nice one, will patch that :100: